runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -85,13 +85,7 @@
 	private static final Pattern RAIDS_PATTERN = Pattern.compile("Your completed (.+) count is: <col=ff0000>(\\d+)</col>.");
 	private static final Pattern WINTERTODT_PATTERN = Pattern.compile("Your subdued Wintertodt count is: <col=ff0000>(\\d+)</col>.");
 	private static final Pattern BARROWS_PATTERN = Pattern.compile("Your Barrows chest count is: <col=ff0000>(\\d+)</col>.");
-	private static final String TOTAL_LEVEL_COMMAND_STRING = "!total";
-	private static final String PRICE_COMMAND_STRING = "!price";
-	private static final String LEVEL_COMMAND_STRING = "!lvl";
-	private static final String CLUES_COMMAND_STRING = "!clues";
-	private static final String KILLCOUNT_COMMAND_STRING = "!kc";
-	private static final String CMB_COMMAND_STRING = "!cmb";
-
+	private static final Pattern COMMAND_PATTERN = Pattern.compile("^[!.]((?<lvl>Lvl|Level)|(?<ge>Ge|Price|Value)|(?<cmb>Cmb|Combat|Cb)|(?<total>Total|Overall|Totallvl|Totallevel)|(?<clues>Clues|Clue)|(?<kc>Kc|Kills|Killcount)) ?(?<txt>.+)?");
```

Comment:
1. This should be broken into a line per command probably.
2. It should also be compiled with `Pattern.CASE_INSENSITIVE`

Reply:
Changes committed.